awx/main/tests/unit/commands/test_dump_auth_config.py

```diff
@@ -0,0 +1,80 @@
+from io import StringIO
+import json
+from django.core.management import call_command
+from django.test import TestCase, override_settings
+
+
+# SAML
+@override_settings(SOCIAL_AUTH_SAML_SP_ENTITY_ID="SP_ENTITY_ID")
+@override_settings(SOCIAL_AUTH_SAML_SP_PUBLIC_CERT="SP_PUBLIC_CERT")
+@override_settings(SOCIAL_AUTH_SAML_SP_PRIVATE_KEY="SP_PRIVATE_KEY")
+@override_settings(SOCIAL_AUTH_SAML_ORG_INFO="ORG_INFO")
+@override_settings(SOCIAL_AUTH_SAML_TECHNICAL_CONTACT="TECHNICAL_CONTACT")
+@override_settings(SOCIAL_AUTH_SAML_SUPPORT_CONTACT="SUPPORT_CONTACT")
+@override_settings(SOCIAL_AUTH_SAML_SP_EXTRA="SP_EXTRA")
+@override_settings(SOCIAL_AUTH_SAML_SECURITY_CONFIG="SECURITY_CONFIG")
+@override_settings(SOCIAL_AUTH_SAML_EXTRA_DATA="EXTRA_DATA")
+@override_settings(SOCIAL_AUTH_SAML_ENABLED_IDPS="ENABLED_IDPS")
+
+# LDAP
+@override_settings(AUTH_LDAP_1_SERVER_URI=["SERVER_URI"])
+@override_settings(AUTH_LDAP_1_BIND_DN="BIND_DN")
+@override_settings(AUTH_LDAP_1_BIND_PASSWORD="BIND_PASSWORD")
+@override_settings(AUTH_LDAP_1_GROUP_SEARCH=["GROUP_SEARCH"])
+@override_settings(AUTH_LDAP_1_GROUP_TYPE="string object")
```

Comment:
lets override this to the actual object this way we can make sure that our output get the object type name rather than the object itself 
@jessicamack